app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/LocalLibraryFragment.kt

```diff
@@ -139,6 +148,60 @@ class LocalLibraryFragment : BaseFragment() {
     go_to_downloads_button_no_files.setOnClickListener {
       offerAction(FileSelectActions.UserClickedDownloadBooksButton)
     }
+
+    select_file.setOnClickListener {
+      showFileChooser()
+    }
+  }
+
+  private fun showFileChooser() {
+    val intent = Intent(Intent.ACTION_GET_CONTENT)
+    intent.type = "*/*"
+    intent.addCategory(Intent.CATEGORY_OPENABLE)
+    try {
+      startActivityForResult(
```

Comment:
[nit] We can use the `apply` operator of kotlin.

Reply:
done